h2o-py/h2o/model/model_base.py

```diff
@@ -1139,6 +1153,9 @@ def varimp_plot(self, num_of_features=None, server=False):
         elif self._model_json["algo"] == "deeplearning":
             plt.title("Variable Importance: H2O Deep Learning", fontsize=20)
             if not server: plt.show()
+        elif self._model_json["algo"] == "glm":
+            plt.title("Variable Importance: H2O GLM", fontsize=20)
+            if not server: plt.show()            
         else:
             raise H2OValueError("A variable importances plot is not implemented for this type of model")
```

Comment:
Need to add this so that GLM will be recognized to be an algo with variable importance.